guide/english/certifications/javascript-algorithms-and-data-structures/object-oriented-programming/use-an-iife-to-create-a-module/index.md

```diff
@@ -45,3 +45,21 @@ let funModule = (function() {
 })();
 
 ```
+
+### Solution 2
+
+If using ES6, the same can be rewritten as:
+
+```javascript
```

Comment:
```suggestion
```js
```

Reply:
@thecodingaviator It is not necessary to use `js`.  `javascript` should work with prism just fine.

Reply:
Agreed but since we're adding it to the other file, I figured why not do it here too?

Reply:
JS works so I'm going to leave this.